src/app/globals.css

```diff
@@ -236,3 +265,21 @@
     var(--utrecht-link-focus-text-decoration-thickness)
   );
 }
+
```

Comment:
In de #284 PR is dit ook opgenomen en die oplossing lijkt niet voor een kleine layout shift te zorgen, het lijkt mij dus verstandig om daarmee verder te gaan.